src/components/DatePicker.module.css

```
@@ -5,3 +5,7 @@
   gap: 0.8rem;
   width: 100%;
 }
+
+.datePicker :hover {
```

```suggestion
.datePicker:hover {
```
Because :hover is a pseudo-class of dataPiker you can remove this space in between them.